common/src/main/java/com/microsoft/hydralab/performance/IPerformanceListener.java

```diff
@@ -0,0 +1,13 @@
+package com.microsoft.hydralab.performance;
+
+public interface IPerformanceListener {
```

Comment:
What's the meaning of PerformanceListener? Is it a listener of performance?

Reply:
Yes, it's a listener of performance testing.

Reply:
Rename the `IPerformanceListener` to `PerformanceTestListener` after offline-sync.